File: Patrones/Estructurales/Composite/Python/Trabajador.py
from Empleado import Empleado

class Trabajador(Empleado):
    def __init__(self, nombre, puesto):
        self.__nombre = nombre
        self.__puesto = puesto

    def mostrarDetalles(self):
        print("Trabajador: " + self.__nombre + ", Puesto: " + self.__puesto)
        print()

File: Patrones/Estructurales/Composite/Java/src/Trabajador.java
public class Trabajador implements Empleado {
    private final String nombre;
    private final String puesto;

    public Trabajador(String nombre, String puesto) {
        this.nombre = nombre;
        this.puesto = puesto;
    }

    @Override
    public void mostrarDetalles() {
        System.out.println("Trabajador: " + nombre + ", Puesto: " + puesto);
    }
}
